### PR TITLE
small changes

### PR DESCRIPTION
**Pull request by zeff9801**

### CODE REVIEW

**Review by evernife — APPROVED** [review]
Seems ok

File: Grimoire-mc1.12.2/src/main/java/io/github/crucible/grimoire/mc1_12_2/api/integration/eventhelper/IEventHelperIntegration.java [hunk 1]
```diff
@@ -23,22 +23,22 @@
 
 public interface IEventHelperIntegration extends IModIntegration {
 
-    public boolean canBreak(@Nonnull EntityPlayer player, @Nonnull BlockPos pos);
+    boolean canBreak(@Nonnull EntityPlayer player, @Nonnull BlockPos pos);
```
Comment (Aizistral) [inline Grimoire-mc1.12.2/src/main/java/io/github/crucible/grimoire/mc1_12_2/api/integration/eventhelper/IEventHelperIntegration.java:26]: Could you please not strip away `public` modifier from interface/annotation methods? I know they aren't necessary, but I prefer them being in place for aesthetical reasons.

File: Grimoire-shared/src/main/java/io/github/crucible/omniconfig/api/properties/IAbstractProperty.java [hunk 2]
```diff
@@ -14,31 +14,31 @@
  * @author Aizistral
  */
 
-public abstract interface IAbstractProperty {
```
Comment (Aizistral) [inline Grimoire-shared/src/main/java/io/github/crucible/omniconfig/api/properties/IAbstractProperty.java:17]: Also this. I know all interfaces are inherently abstract, but I mark them as such for the purpose of indication that nothing implements this interface on its own, only interfaces that extend this one.

File: Grimoire-shared/src/main/java/io/github/crucible/omniconfig/api/core/VersioningPolicy.java [hunk 3]
```diff
@@ -67,5 +67,5 @@
      * action if config versions do not match. It will simply update config version
      * to defined one upon saving the file, and leave anything else as is.
      */
-    DISMISSIVE;
```
Comment (Aizistral) [inline Grimoire-shared/src/main/java/io/github/crucible/omniconfig/api/core/VersioningPolicy.java:70]: Plase leave these. Though not neccessary for enums without constructor, there might be constructor someday.

File: Grimoire-shared/src/main/java/io/github/crucible/omniconfig/api/builders/BuildingPhase.java [hunk 4]
```diff
@@ -27,7 +27,7 @@
      */
     FINALIZATION;
 
-    private BuildingPhase() {
```
Comment (Aizistral) [inline Grimoire-shared/src/main/java/io/github/crucible/omniconfig/api/builders/BuildingPhase.java:30]: Same as for interface methods modifier.

File: Grimoire-shared/src/main/java/io/github/crucible/grimoire/common/core/MixinConfiguration.java [hunk 5]
```diff
@@ -117,7 +117,7 @@ public boolean isRuntimeGenerated() {
 
     @Override
     public boolean canLoad() {
-        return !this.isLoaded && this.isRuntimeGenerated() ? ConfigBuildingManager.areRuntimeConfigsGenerated() : true;
+        return this.isLoaded || !this.isRuntimeGenerated() || ConfigBuildingManager.areRuntimeConfigsGenerated();
```
Comment (Aizistral) [inline Grimoire-shared/src/main/java/io/github/crucible/grimoire/common/core/MixinConfiguration.java:120]: How the hell this is supposed to work well?

File: Grimoire-shared/src/main/java/io/github/crucible/grimoire/common/api/integration/IModIntegrationRegistry.java [hunk 6]
```diff
@@ -29,20 +29,19 @@
      * integration.
      * @return Integration instance if exists, null if does not.
      */
-    @Nullable
-    public <T extends IModIntegration> T getIntegration(Class<T> integrationClass);
+    @Nullable <T extends IModIntegration> T getIntegration(Class<T> integrationClass);
```
Comment (Aizistral) [inline Grimoire-shared/src/main/java/io/github/crucible/grimoire/common/api/integration/IModIntegrationRegistry.java:32]: No, you don't put annotation and its target methods in the same line. Please, for the love of gods.

File: Grimoire-shared/src/main/java/io/github/crucible/grimoire/common/api/eventbus/CoreEvent.java [hunk 7]
```diff
@@ -112,7 +112,7 @@ public void setResult(Result result) throws UnsupportedOperationException {
      *
      * @author Aizistral
      */
-    public static enum Result {
```
Comment (Aizistral) [inline Grimoire-shared/src/main/java/io/github/crucible/grimoire/common/api/eventbus/CoreEvent.java:115]: Why?

**Review by Aizistral — CHANGES_REQUESTED** [review]
No way I'll let this be merged as-is.

**Comment by evernife** [comment]
If he revert all that, it will be reverting like more than 70% of his changes.

What he did was just follow standarts.

image

_(Maybe this changes were automatic)_

**Comment by Aizistral** [comment]
> _(Maybe this changes were automatic)_

Most probably. Redundant modifiers are not "specifying them is an error" modifiers, they are "you don't have to specify them, but if you wanna - go nuts" modifiers.  I find the code easier to perceive and navigate when they are in place. which is by no measure less important than just having that code doing its functional job.

**Comment by Aizistral** [comment]
I mean, you don't have to care as much if you only touch upon internals, or introduce some new feature to the API. In latter case I can always make corrections myself if something purely visual annoys me. But if you're gonna auto-refractor everything you didn't write yourself, you could at least bother enough to ask which of your intended changes would be welcome first.

**Comment by evernife** [comment]
> I mean, you don't have to care as much if you only touch upon internals, or introduce some new feature to the API. In latter case I can always make corrections myself if something purely visual annoys me. But if you're gonna auto-refractor everything you didn't write yourself, you could at least bother enough to ask which of your intended changes would be welcome first.

I know exactly how you feel :D

**Comment by Aizistral** [comment]
Since it's been over a month and no response followed from author of the PR, I am closing this.

**Pull request closed without merging**